pkgs/applications/science/math/R/default.nix

```diff
@@ -49,10 +52,22 @@ stdenv.mkDerivation rec {
       LDFLAGS="-L${gfortran.cc}/lib"
       RANLIB=$(type -p ranlib)
       R_SHELL="${stdenv.shell}"
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+      --without-tcltk
+      --disable-R-framework
+      CC="clang"
+      CXX="clang"
+      OBJC="clang"
+  '' + ''
     )
     echo "TCLLIBPATH=${tk}/lib" >>etc/Renviron.in
   '';
 
+  postConfigure = stdenv.lib.optionalString stdenv.isDarwin ''
+    sed -i 's|/usr/share/zoneinfo|${tzdata}/share/zoneinfo|g' src/library/base/R/datetime.R
```

Comment:
Why is that patch required only on Darwin? `/usr/share/zoneinfo` doesn't exist on NixOS either, so why isn't this patch required on Linux, too?